docs/Data_Submission_Portal/Users_Guide/Troubleshooting.md

```diff
@@ -0,0 +1,1236 @@
+                        "keys":["type"],
+                        "message":"missing 'type'",
+                        "type":"INVALID_TYPE"
+                    },
+                    {
+                        "keys":["type"],
+                        "message":"'type' is a required property",
+                        "type":"MISSING_PROPERTY"
+                    }
+                ]
+            "id":
+                [
+                    {
+                        "id":null
+                    }
+                ],
+            "related_cases":[],
+            "type":null,
+            "unique_keys":[],
+            "valid":false,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":1,
+    "message":"Transaction aborted due to 1 invalid entity.",
+    "success":false,
+    "transaction_id":6372998,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":0
+}
+```
+```Request2
+{
+  "projects": {
+    "code": "INTERNAL"
+  }, 
+  "disease_type": "Adenomas and Adenocarcinomas", 
+  "primary_site": "Bladder",
+  "project_id": "GDC-INTERNAL", 
+  "type": "case",
+  "submitter_id": "GDC-INTERNAL-000099"
+}
+```
+```Response2
+{
+  "cases_related_to_created_entities_count": 0,
+  "cases_related_to_updated_entities_count": 0,
+  "code": 200,
+  "created_entity_count": 0,
+  "entities":
+    [
+        {
+            "action":"update",
+            "errors":[],
+            "id":
+                [
+                    {
+                        "id":"f0c68d72-ee54-4ad9-89e8-b08398b5aa0b"
+                    }
+                ],
+            "related_cases":[],
+            "type":"case",
+            "unique_keys":
+                [
+                    {
+                        "project_id":"GDC-INTERNAL",
+                        "submitter_id":"GDC-INTERNAL-000099"
+                    }
+                ],
+            "valid":true,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":0,
+    "message":"Transaction successful.",
+    "success":true,
+    "transaction_id":6372999,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":1
+}
+```
+
+#### NOT_FOUND Messages
+
+|Message|Explanation|Solution|
+| --- | --- | --- |
+| __Cannot create node with new submitter id specified. If update action (PUT) requested, Ensure specified id/submitter id exists__ | The submitter_id does not exist so the entity cannot be versioned and replaced with an entity with the new_submitter_id | If the new_submitter_id already exists and the md5sum is the same, this entity has already been uploaded and can be skipped. If neither the submitter_id nor the new_submitter_id exists, remove submitter_id and change "new_submitter_id" to "submitter_id" |
+| __Unable to validate case against dbGaP. {}__ | The case submitter_id is not registered in dbGaP | Ensure that there are no typos in the submitter_id or submit additional cases to dbGaP |
```

Comment:
For these two, try uploading a case with a  nonsense submitter_id.  Keep whichever error shows up and remove the other other.  I think one of them might indicate that dbGaP went down or something.